tests/Database/Table/bugs/Selection.emptyResultSet.phpt

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @dataProvider? ../../databases.ini
+ */
+
+use Tester\Assert;
+
+require __DIR__ . '/../../connect.inc.php'; // create $connection
+
+Nette\Database\Helpers::loadFromFile($connection, __DIR__ . "/../../files/{$driverName}-nette_test1.sql");
+
+
+test(function() use ($context) {
+  $selection = $context->table('book');
```

Comment:
please use uppercased `NULL`, single quotes where possible and space in `function ()`.